### ISSUE TITLE
ptvs - search solution

### ISSUE THREAD

**Issue by denfromufa** [Microsoft/PTVS#796]
I imported pandas as a solution to PTVS and then wanted to search for one method, but the **"Search Solution Explorer (Ctrl+;)"** only works for file names. I believe in C# I can search .cs file contents as well.

I can do Ctrl-F and find all, but this tree view is more convenient.

**Comment by int19h** [comment]
You can also use Quick Search for this (the default shortcut is Ctrl+, if I remember correctly) - this will do a search on identifiers, rather than a plain text grep.

To enable search in Solution Explorer, we need to enable general integration with it - i.e. like in C#, where you can expand a .cs file in there and see all classes as children. It was this item in the old tracker: http://pytools.codeplex.com/workitem/300

**Comment by denfromufa** [comment]
thanks for linking the old issue. Ctrl+, is not good :-1:  - not even sure what it is searching for?!

**Comment by int19h** [comment]
Quick Search _should_ be searching for files throughout the solution, as well as various identifiers (modules, classes, functions etc). For PTVS, it is implemented on top of the same code that populates Object Browser, so if you see something there, you should also be able to find it via QS.

**Comment by denfromufa** [comment]
yes, objects browser and QS are both working now. Looks like previous time the parsing has not finished yet (pandas is huge).

**Comment by zooba** [comment]
Closing this - we're all too aware of the slow analysis of large projects. :(

**Comment by denfromufa** [comment]
@zooba I think the original issue here is supporting  "Search Solution Explorer (Ctrl+;)" semantically inside all .py files in solution like in C#. there is even linked issue from codeplex. not sure why closing?

**Comment by zooba** [comment]
You're right, I missed that.
